client/my-sites/domains/domain-management/list/bulk-all-domains.tsx

```diff
@@ -40,150 +40,149 @@ export default function BulkAllDomains( props: BulkAllDomainsProps ) {
+				// Add border around everything
+				overflow: hidden;
+				min-height: 100vh;
+				padding: 16px 16px 16px calc( var( --sidebar-width-max ) );
 
-					.layout_primary > main {
-						padding-bottom: 0;
-					}
+				.layout_primary > main {
+					padding-bottom: 0;
 				}
+			}
 
-				.layout__secondary .global-sidebar {
-					border: none;
-				}
+			.layout__secondary .global-sidebar {
+				border: none;
+			}
 
-				.has-no-masterbar .layout__content {
-					padding-top: 16px !important;
-				}
+			.has-no-masterbar .layout__content {
+				padding-top: 16px !important;
+			}
 
-				header.navigation-header {
-					padding-top: 24px;
+			header.navigation-header {
+				padding-top: 24px;
 
-					.formatted-header {
-						max-height: 41px;
-					}
+				.formatted-header {
+					max-height: 41px;
+				}
 
-					.formatted-header__title {
-						color: var( --studio-gray-80, #2c3338 );
-						font-family: 'SF Pro Display', sans-serif;
-						font-size: 1.5rem;
-						font-style: normal;
-						font-weight: 500;
-						line-height: 1.2;
-					}
-					.domain-header__buttons .button {
-						border-radius: 4px;
-						white-space: nowrap;
-						margin-left: 0;
-						&:not( .is-primary ) {
-							margin-inline-end: 1rem;
-						}
+				.formatted-header__title {
+					color: var( --studio-gray-80, #2c3338 );
+					font-family: 'SF Pro Display', sans-serif;
+					font-size: 1.5rem;
+					font-style: normal;
+					font-weight: 500;
+					line-height: 1.2;
+				}
+				.domain-header__buttons .button {
+					border-radius: 4px;
+					white-space: nowrap;
+					margin-left: 0;
+					&:not( .is-primary ) {
+						margin-inline-end: 1rem;
 					}
 				}
+			}
 
-				.search-component.domains-table-filter__search.is-open.has-open-icon {
-					border-radius: 4px;
-					height: 44px;
+			.search-component.domains-table-filter__search.is-open.has-open-icon {
+				border-radius: 4px;
+				height: 44px;
+			}
+
+			.domains-table {
+				margin-top: 48px;
+				.domains-table-toolbar {
+					margin-inline: 64px;
 				}
+				table {
+					overflow-y: auto;
+					max-height: calc( 100vh - 235px );
+					padding-inline: 64px;
+					margin-bottom: 0;
+				}
+				.domains-table-header {
+					position: sticky;
+					top: 0;
+					z-index: 2;
+				}
+			}
 
-				.domains-table {
-					margin-top: 48px;
-					.domains-table-toolbar {
-						margin-inline: 64px;
-					}
-					table {
-						overflow-y: auto;
-						max-height: calc( 100vh - 235px );
+			@media only screen and ( min-width: 782px ) {
+				div.layout.is-global-sidebar-visible {
+					header.navigation-header {
+						padding-top: 24px;
 						padding-inline: 64px;
-						margin-bottom: 0;
+						border-block-end: 1px solid var( --studio-gray-0 );
 					}
-					.domains-table-header {
-						position: sticky;
-						top: 0;
-						z-index: 2;
+					.layout__primary > main {
+						background: var( --color-surface );
+						border-radius: 8px;
+						box-shadow: 0px 0px 17.4px 0px rgba( 0, 0, 0, 0.05 );
```

Comment:
Note - this is the only change I made in this file.  Previously it noted `box-shadow: none` and was missing the `;` breaking the next line style as well. Adding the `;` and saving caused the entire file to auto-format indentation and make these changes look much bigger.